src/consensus/params.cpp

```diff
@@ -12,6 +12,46 @@
 #include "util/match.h"
 
 namespace Consensus {
+    /**
+     * General information about each funding stream.
+     * Ordered by Consensus::FundingStreamIndex.
+     */
+    constexpr struct FSInfo FundingStreamInfo[Consensus::MAX_FUNDING_STREAMS] = {
+        {
+            .recipient = "Electric Coin Company",
+            .specification = "https://zips.z.cash/zip-0214",
+            .valueNumerator = 7,
+            .valueDenominator = 100,
+        },
+        {
+            .recipient = "Zcash Foundation",
+            .specification = "https://zips.z.cash/zip-0214",
+            .valueNumerator = 5,
+            .valueDenominator = 100,
+        },
+        {
+            .recipient = "Major Grants",
+            .specification = "https://zips.z.cash/zip-0214",
+            .valueNumerator = 8,
+            .valueDenominator = 100,
+        },
+        {
+            .recipient = "Lockbox",
+            .specification = "https://zips.z.cash/draft-nuttycom-funding-allocation",
+            .valueNumerator = 20,
+            .valueDenominator = 100,
```

Comment:
We haven't yet chosen between https://zips.z.cash/draft-nuttycom-funding-allocation and https://zips.z.cash/draft-zf-community-dev-fund-2-proposal